flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -648,28 +648,30 @@ public void selectionCleared() {
     FlutterAllMemoryData.ThreadSafeData rssDataSeries = stage.getRSSDataSeries();
 
     Range dataRanges = new Range(0, 1024 * 1024 * 100);
+    Range viewRange = getTimeline().getViewRange();
+
     RangedContinuousSeries usedMemoryRange =
-      new RangedContinuousSeries("Memory", getTimeline().getViewRange(), dataRanges, memoryUsedDataSeries);
+      new RangedContinuousSeries("Used", viewRange, dataRanges, memoryUsedDataSeries);
     RangedContinuousSeries maxMemoryRange =
-      new RangedContinuousSeries("MemoryMax", getTimeline().getViewRange(), dataRanges, memoryMaxDataSeries);
+      new RangedContinuousSeries("Capacity", viewRange, dataRanges, memoryMaxDataSeries);
     RangedContinuousSeries externalMemoryRange =
-      new RangedContinuousSeries("MemoryExtern", getTimeline().getViewRange(), dataRanges, memoryExternalDataSeries);
-    // TODO(terry): Temporary comment out working on displaying RSS properly in chart.
-    // RangedContinuousSeries rssRange = new RangedContinuousSeries("RSS",
-    //                                                              getTimeline().getViewRange(), dataRanges, rssDataSeries);
+      new RangedContinuousSeries("External", viewRange, dataRanges, memoryExternalDataSeries);
+    RangedContinuousSeries rssRange =
+      new RangedContinuousSeries("RSS", viewRange, dataRanges, rssDataSeries);
```

Comment:
Add a TODO to add a tooltip explaining what RSS is and why users should care. Perhaps the matching Android code already has a tooltip. I expect the term RSS used here is very confusing to users who will think of RSS feeds instead of Resident Set Size. @InMatrix any thoughts on what term our users might understand better. The goal for showing RSS is to indicate how close an app is to getting shutdown for using more memory than the device allows.
https://en.wikipedia.org/wiki/Resident_set_size

Reply:
I could change the name in the legend to Resident Set Size

Reply:
A tooltip explaining Resident Set Size would be useful.

Reply:
I'm going to hide RSS for now (under debug flag) will re-visit after talking to Siva about why he cares so much about RSS for Flutter devs.